project/CONTRIBUTING.md

```diff
@@ -299,3 +300,42 @@ This is a consensus made on #4242 in addition to #1337.
 We agreed that blank color is boring and so deja vu. Pink is the new way to do.
 ```
 (Obviously, this commit is fake. :wink:)
+
+## Drafting a new release
+
+### Minor releases and patch releases
+
+Releasing software is the last step to getting your bugfixes or new features to your user base,
+and should be done regularly, so that users are not tempted to use development branches.
+To know what you are going to release on branch 42.x, given that the last release on this branch is 42.3.1,
+go to `https://github.com/sonata-project/SonataCoreBundle/compare/42.3.1...42.x`.
+You should see a list of commits, some of which should contain links to pull requests.
+
+#### Determining the next release number
+
+If any of those pull requests is labeled `minor`, then the next release should be a minor release (42.4.0).
+In the opposite case, and if any of those pull requests is labeled `patch`,
+then the next release should be a patch release (42.3.2).
+If none of the conditions above are fulfilled, then all the commits should be labeled either `docs` or `pedantic`,
+and you should not make a release.
```

Comment:
maybe like this:


If there are neither minor noch patch pull requests, all the others should be labelled docs or pedantic, and you should not make a release.

Note: Major changes have to go to the master branch, not the stable branch.